tools/autograd/gen_autograd.py

```diff
@@ -55,12 +55,16 @@ def gen_autograd(
     # Generate VariableType.h/cpp
     from .gen_trace_type import gen_trace_type
     from .gen_variable_type import gen_variable_type
+    from .gen_inplace_or_view_type import gen_inplace_or_view_type
     if not disable_autograd:
         gen_variable_type(out, native_functions_path, fns_with_diff_infos, template_path)
 
         # operator filter not applied as tracing sources are excluded in selective build
         gen_trace_type(out, native_functions_path, template_path)
 
+        gen_inplace_or_view_type(out, native_functions_path, fns_with_diff_infos, template_path)
```

Comment:
this is very minor, but if you're back in here I'd suggest putting this right after the call to `gen_variable_type()`, just to visually reinforce the fact that they're related.